### PR TITLE
Manage lvm2 package

### PR DESCRIPTION
**Pull request by rhoml**
Reopening https://github.com/puppetlabs/puppetlabs-lvm/pull/71

### CODE REVIEW

**Comment by hunner** [comment]
Thanks! Could you squash this down to a single commit? Also, I know this PR is just for Debian and Ubuntu, but do you know if this applies to RedHat family OSs also? lvm2 is a package on at least rhel.

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,5 @@
 .*.swp
 Gemfile.lock
 .bundle/
+/vendor/
+/bin/
```
Comment (hunner) [inline .gitignore]: Could you not commit the .gitignore? Thanks.

**Comment by hunner** [comment]
I'm curious... lvm2 appears to be a base level package and always present. Under what circumstances would this not be present?

**Comment by rhoml** [comment]
@hunner Rebase is done, since redhat also have lvm2 packages I figured out to create a flag to manage lvm2 package with the module or not depending on people needs.

**Comment by rhoml** [comment]
@hunner  any chance that this can get merged? :)

**Pull request merged**